README.md

```diff
@@ -1,9 +1,9 @@
 # val3dity 
 
 Validation of 3D primitives according to the international standard ISO19107.
-Think of it as [PostGIS ST_IsValid](http://postgis.net/docs/ST_IsValid.html), but for 3D primitives (PostGIS is only for 2D ones).
+Think of it as [PostGIS ST_IsValid](http://postgis.net/docs/ST_IsValid.html), but for 3D primitives (PostGIS only validates 2D primitives).
 
-It allows us to validate a 3D primitive, ie to verify whether it respects the definition as given in [ISO19107](http://www.iso.org/iso/catalogue_detail.htm?csnumber=26012) and GML/CityGML.
```

Comment:
I prefer to write "ie" and "eg" without the dots... I follow the Guardian Style Guide: https://www.theguardian.com/guardian-observer-style-guide-e